src/draw/sw/blend/neon/lv_blend_neon.S

```diff
@@ -335,14 +335,14 @@ TMP_Q1      .qn  q14
     vmov.u8         OPA, #0xFF
 .endif
 
-    sub             DST_STRIDE, DST_STRIDE, DST_W
     vmvn            D_A, OPA
 .if dst_bpp == 16
-    lsl             DST_STRIDE, DST_STRIDE, #1
+    sub             DST_STRIDE, DST_STRIDE, DST_W, lsl #1
 .elseif dst_bpp == 24
-    add             DST_STRIDE, DST_STRIDE, DST_STRIDE, lsl #1
+    sub             DST_STRIDE, DST_STRIDE, DST_W
+    sub             DST_STRIDE, DST_STRIDE, DST_W, lsl #1
 .elseif dst_bpp >= 31
```

Comment:
Is this a typo?

Reply:
no :)

Reply:
`31` means for XRGB8888, and 32 for ARGB8888. So it's not a typo.
See 
https://github.com/lvgl/lvgl/blob/e4de2d265aa1b251038d18659835502e95d14eaf/src/draw/sw/blend/neon/lv_blend_neon.S#L681-L682